framework/Yii.php

```diff
@@ -1,7 +1,5 @@
 <?php
 /**
- * Yii bootstrap file.
```

Comment:
File header should be consistent across all files - this is only place with some extra description. The only way to avoid this is to add this file to exceptions list - php-cs-fixer will not check this file at all.